tshirts.py

```diff
@@ -1,14 +1,22 @@
-
 def size(cms):
-    if cms < 38:
+    if cms not in range(15,50): # if shoulder size is out of range 
+        return None
+    elif cms <= 38:
         return 'S'
-    elif cms > 38 and cms < 42:
+    elif cms > 38 and cms <= 42:
         return 'M'
     else:
         return 'L'
 
 
 assert(size(37) == 'S')
+assert(size(38) == 'S')
+assert(size(5) == None)  # testcase for invalid scenario
+assert(size(38) == 'L')

```

Comment:
the tests expect the production code to classify size 38 as both S and L. Is that possible? also for 42...